blueocean-dashboard/src/main/js/components/karaoke/components/Step.jsx

```diff
@@ -27,14 +28,8 @@ function createStepLabel(step) {
 export class Step extends Component {
     constructor(props) {
         super(props);
-        const { augmenter, step } = props;
```

Comment:
Another thing is to get rid of `Augmenter` what the heck is an augmenter? It just stores properties about the current view of the pipeline, basically...